Makefile

```diff
@@ -938,7 +938,111 @@ ifeq "$(OPENMP)" "true"
 endif
 
 
-pio_test:
+openacc_test:
+ifeq "$(OPENACC)" "true"
+	@#
+	@# First ensure that both FFLAGS_ACC and CFLAGS_ACC are not blank
+	@# If these are not set for a target, then OpenACC most likely cannot compile
+	@#
+	@echo "Checking if FFLAGS_ACC and CFLAGS_ACC are defined for [$(BUILD_TARGET)]..."
+	@if ([ -z "$(FFLAGS_ACC)" ] && [ -z "$(CFLAGS_ACC)" ]); then \
+	   (echo "*********************************************************"; \
+	    echo "ERROR: OPENACC=true was specified, but [$(BUILD_TARGET)] build target does not seem to support OpenACC:"; \
+	    echo "    FFLAGS_ACC and CFLAGS_ACC are both undefined for [$(BUILD_TARGET)] in the top-level Makefile."; \
+	    echo "Please set these variables to appropriate OpenACC compilation flags in the [$(BUILD_TARGET)] target to enable OpenACC support."; \
+	    echo "*********************************************************"; exit 1)\
+	else \
+	   echo "=> FFLAGS_ACC or CFLAGS_ACC defined"; \
+	fi
+
+	@#
+	@# Create test C and Fortran programs that look for OpenACC header file and parallelize a loop
+	@#
+	@printf "#include <openacc.h>\n\
+	        &int main(){\n\
+	        &    int n_devs=acc_get_num_devices( acc_device_default );\n\
+	        &    int i,n=0;\n\
+	        &    #pragma acc kernels\n\
+	        &    for (i=0; i<10; i++)\n\
+	        &      n=n+i;\n\
+	        &    return 0;\n\
+	        &}\n" | sed 's/^[ \t]*&//' > openacc.c
```

Comment:
When indenting test programs with tab followed by spaces, I think we can omit the `\t` in the `sed` commands.
From the GNU Make manual on [Splitting Recipe Lines](https://www.gnu.org/software/make/manual/make.html#Splitting-Recipe-Lines):

> However, in contrast to how they are treated in other places in a makefile (see [Splitting Long Lines](https://www.gnu.org/software/make/manual/make.html#Splitting-Lines)), backslash/newline pairs are not removed from the recipe. Both the backslash and the newline characters are preserved and passed to the shell. How the backslash/newline is interpreted depends on your shell. If the first character of the next line after the backslash/newline is the recipe prefix character (a tab by default; see [Special Variables](https://www.gnu.org/software/make/manual/make.html#Special-Variables)), then that character (and only that character) is removed. Whitespace is never added to the recipe.

Reply:
I agree, the `\t` can be ommited. When I was playing around with this sed command I figured out what the documentation said through trial and error.